apps/console/src/index.tsx

```diff
@@ -36,6 +38,9 @@ import Theme from "./theme";
 // Set the runtime config in the context.
 ContextUtils.setRuntimeConfig(Config.getDeploymentConfig());
 
+// Set the monaco editor loader config.
```

Comment:
```suggestion
// Manually load the `monaco` editor instance to use as an npm package instead of using CDN.
// This enables the usage of the editor instance in offline mode.
```